src/main/java/com/github/princesslana/eriscasper/ErisCasper.java

```diff
@@ -57,7 +68,10 @@ public void run(Bot bot) {
   }
 
   public static ErisCasper create() {
-    return create(System.getenv("EC_TOKEN"));
+    return create(
+        CONFIG
+            .get("ec.token")
```

Comment:
How does this work? Should we throw in a default configuration file for example. I think this is a bit ambiguous as to what to do. Maybe throw a config file into the resources and if not built create a default one for them. Something needs to be done so before installation there's a way to figure out how to set it up, rather that be in docs or some other example.

Reply:
This doesn't read from files yet, it just adds system properties as per #106. So you can configure the token by setting the `EC_TOKEN` environment variable, or the `ec.token` system property.

We do need a getting started guide, but imo that's outside the scope of this issue.

Reply:
Ah okay, so this is just the beginning to implementing a configuration file, or rather the implementation to provide support for it.

Reply:
It's the beginning of enabling configuration to happen in ways other than just environment variables, yes.

Reply:
Then this is probably good for now, it definitely needs work in documentation and examples for installation. I'll put that in the issue list.

Reply:
#108 exists for documentation

Reply:
I don't see configuration specification. Should we add that as a checklist item.

Reply:
Or leave a comment

Reply:
Alright, I'll leave a comment there and accept this PR through.